src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Ball ball = new Ball();
+        Random random = new Random();
+        final int BO = 100;
```

Comment:
move this to the class constant
![image](https://user-images.githubusercontent.com/12864128/191986801-0b335228-e4dc-4530-89dc-60be0dc568a2.png)